templates/includes/jumplinks.html

```diff
@@ -2,13 +2,13 @@
     <nav class="jumplinks">
         <div class="container">
             <h2 class="jumplinks__heading">On this page</h2>
-            <ol class="jumplinks__list">
+            <ul class="jumplinks__list">
```

Comment:
I would have thought an ordered list was a more suitable choice from a semantics perspective, since it's an ordered series?

Reply:
Yes, very true. I was copying a bit too blindly. Thank you 👍